profiler/src/ProfilerEngine/Datadog.Profiler.Native/RawSample.h

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include <stdint.h>
+#include <vector>
+#include "cor.h"
+#include "corprof.h"
+#include "ManagedThreadInfo.h"
+
+
+class RawSample
+{
+public:
+    RawSample();
+    // no need to define a move-operator because it would be equivalent to the compiler-generated copy constructor
+    // i.e. no field contains deep copiable object (it would have been different if vector<string> for example
```

Comment:
you can remove this comment.
By default a move ctor is generated by the compiler. This comment would add confusion instead.

Reply:
✅